string aruco_id
---
bool sent_goal